sdks/cpp/sdk.cc

```diff
@@ -59,6 +59,18 @@ namespace agones {
         return stub->GetGameServer(context, request, response);
     }
 
+    grpc::Status SDK::WatchGameServer(const std::function<void(stable::agones::dev::sdk::GameServer)> callback) {
```

Comment:
@Kuqd @EricFortin My C++ is terrible - I hope I haven't done anything totally silly here?

Reply:
looks good to me, may be @victor-prodan can confirm ?

Reply:
Yes, looks fine to me as well.